test/lib/Elastica/Test/Facet/DateHistogramTest.php

```diff
@@ -28,22 +28,23 @@ public function testTest()
 
         $mapping = new Mapping($type, array(
                 'name' => array('type' => 'string', 'store' => 'no'),
-                'dtmPosted' => array('type' => 'date', 'store' => 'no', 'format' => 'yyyy-MM-dd HH:mm:ss'),
+                'dtmPosted' => array('type' => 'long', 'store' => 'no'),
```

Comment:
Wouldn't it make sense to keep this test as it is and add a new one? Liek this we now that both options work. Or what is the reason you modified this test?